src/adc/iio/hardware.rs

```diff
@@ -35,10 +35,12 @@ use thread_priority::*;
 
 use crate::measurement::{Measurement, Timestamp};
 
+type ChannelDesc = (&'static str, &'static str, &'static str);
```

Comment:
I would rather introduce a real struct at this point like, which makes it clearer what each string mean.

```rust
struct AdcChannelDesc {
  name: &'static str,
  xxx: &'static str,
  zzz: &'static str,
}
```